analysis/cnvs/parse_prep_normalize_nch_cnvs.ipynb

```diff
@@ -1419,25 +1490,25 @@
     "        \"assembly\": cnv[\"build\"],\n",
     "        \"chromosome\": f\"chr{cnv['chromosome']}\",\n",
     "        \"start0\": int(cnv[\"start\"]),\n",
-    "        \"start_pos_type\": \"Number\",\n",
+    "        \"start_pos_type\": \"number\",\n",
     "        \"end0\": int(cnv[\"stop\"]),\n",
-    "        \"end_pos_type\": \"Number\",\n",
+    "        \"end_pos_type\": \"number\",\n",
     "    }\n",
     "    if (\n",
     "        cnv[\"copy_number_min\"] == cnv[\"copy_number_max\"]\n",
     "    ):  ##if copy number is unambiguous, not a range\n",
     "        args[\"copies0\"] = int(cnv[\"copy_number_min\"])\n",
-    "        args[\"copies_type\"] = \"Number\"\n",
+    "        args[\"copies_type\"] = \"number\"\n",
     "    else:  ##if copy number is a range e.g. 3~4\n",
     "        args[\"copies0\"] = int(cnv[\"copy_number_min\"])\n",
     "        args[\"copies1\"] = int(cnv[\"copy_number_max\"])\n",
-    "        args[\"copies_type\"] = \"DefiniteRange\"\n",
+    "        args[\"copies_type\"] = \"definite_range\"\n",
     "\n",
     "    args[\"do_liftover\"] = True\n",
     "    args[\"untranslatable_returns_text\"] = False\n",
     "\n",
     "    rec = dict(cnv)\n",
-    "    resp = json.loads(parsed_to_cn_var(ParsedToCnVarQuery(**args)).json())\n",
+    "    resp = json.loads(parsed_to_cn_var(ParsedToCnVarQuery(**args)).model_dump_json())\n",
```

Comment:
File still has nulls for GKS objects. Going to leave this as an exercise for you to solve